test/OpenTelemetry.Tests/Logs/LogRecordTest.cs

```diff
@@ -213,7 +214,16 @@ public void CheckStateForStrucutredLogWithGeneralType()
             Assert.Contains(state, item => item.Key == "{OriginalFormat}");
             Assert.Equal("{food}", state.First(item => item.Key == "{OriginalFormat}").Value);
 
-            Assert.Equal("[Name, truffle], [Price, 299.99]", state.ToString());
+            var prevCulture = CultureInfo.CurrentCulture;
```

Comment:
If there is a common need, consider make this a "using statement".

Reply:
also, we can move to shared extension method like https://github.com/open-telemetry/opentelemetry-dotnet/blob/master/src/OpenTelemetry.Api/Internal/ExceptionExtensions.cs#L31

Reply:
This was the only test that failed that way, at least with a German locale.